catalog/nsili/catalog-nsili-endpoint/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -27,6 +27,11 @@
                 default="60"
         />
         <AD
+                description="Maximum time to wait before starting a standing query (seconds)"
```

Comment:
I updated the metatype and blueprint to pass this in minutes instead of seconds.